src/meshapi/models/link.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+
+from meshapi.models.devices.device import Device
+
+
+class Link(models.Model):
+    class LinkStatus(models.TextChoices):
+        INACTIVE = "Inactive"
+        PLANNED = "Planned"
+        ACTIVE = "Active"
+
+    class LinkType(models.TextChoices):
+        STANDARD = "Standard"
+        VPN = "VPN"
+        MMWAVE = "MMWave"
+        FIBER = "Fiber"
```

Comment:
Should we preemptively add some more link types?
- Wired
- 24GHz
- 5GHz
- Etc?

Reply:
Not at this time. Easy to play with in the future